snabbdom/src/test/scala/com/github/buntec/snabbdom/StyleSuite.scala

```diff
@@ -0,0 +1,242 @@
+      val vnode1 = h("i", cachedStyles)
+      val vnode2 = h("i", cachedStyles)
+      val elm1 = patch(vnode0, vnode1).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm1.style.fontSize, "14px")
+      assertEquals(elm1.style.display, "inline")
+      val elm2 = patch(vnode1, vnode2).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm2.style.fontSize, "14px")
+      assertEquals(elm2.style.display, "inline")
+    }
+
+    vnode0.test("updates styles") { vnode0 =>
+      val vnode1 = h(
+        "i",
+        VNodeData.builder
+          .withStyle("fontSize" -> "14px", "display" -> "inline")
+          .build
+      )
+
+      val vnode2 = h(
+        "i",
+        VNodeData.builder
+          .withStyle("fontSize" -> "12px", "display" -> "block")
+          .build
+      )
+
+      val vnode3 = h(
+        "i",
+        VNodeData.builder
+          .withStyle("fontSize" -> "10px", "display" -> "block")
+          .build
+      )
+
+      val elm1 = patch(vnode0, vnode1).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm1.style.fontSize, "14px")
+      assertEquals(elm1.style.display, "inline")
+      val elm2 = patch(vnode1, vnode2).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm2.style.fontSize, "12px")
+      assertEquals(elm2.style.display, "block")
+      val elm3 = patch(vnode2, vnode3).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm3.style.fontSize, "10px")
+      assertEquals(elm3.style.display, "block")
+
+    }
+
+    vnode0.test("explicitly removes styles") { vnode0 =>
+      val vnode1 =
+        h("i", VNodeData.builder.withStyle("fontSize" -> "14px").build)
+      val vnode2 =
+        h("i", VNodeData.builder.withStyle("fontSize" -> "").build)
+      val vnode3 =
+        h("i", VNodeData.builder.withStyle("fontSize" -> "10px").build)
+      val elm1 = patch(vnode0, vnode1).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm1.style.fontSize, "14px")
+      val elm2 = patch(vnode1, vnode2).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm2.style.fontSize, "")
+      val elm3 = patch(vnode2, vnode3).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm3.style.fontSize, "10px")
+    }
+
+    vnode0.test("implicitly removes styles from element") { vnode0 =>
+      val vnode1 =
+        h("i", VNodeData.builder.withStyle("fontSize" -> "14px").build)
+      val vnode2 = h("i", VNodeData.empty)
+      val vnode3 =
+        h("i", VNodeData.builder.withStyle("fontSize" -> "10px").build)
+      val elm1 = patch(vnode0, vnode1).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm1.style.fontSize, "14px")
+      val elm2 = patch(vnode1, vnode2).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm2.style.fontSize, "")
+      val elm3 = patch(vnode2, vnode3).elm.get.asInstanceOf[dom.HTMLElement]
+      assertEquals(elm3.style.fontSize, "10px")
+    }
+
+    vnode0.test("updates css variables") { vnode0 =>
+      if (hasCssVariables) {
```

Comment:
```suggestion
      assume(hasCssVariables)
```
See https://scalameta.org/munit/docs/filtering.html#ignore-single-test-case-based-on-a-dynamic-conditions

Reply:
Cool!